utils/swift_build_support/swift_build_support/products/clang.py

```diff
@@ -0,0 +1,17 @@
+# swift_build_support/products/clang.py -------------------------*- python -*-
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ----------------------------------------------------------------------------
+
+from . import product
+
+
+class Clang(product.Product):
```

Comment:
Do we need separate products for LLVM and Clang?  They are built within the same CMake invocation.